src/froll.c

```diff
@@ -397,6 +397,277 @@ void frollsumExact(double *x, uint64_t nx, ans_t *ans, int k, double fill, bool
   }
 }
 
+/* fast rolling max */
+void frollmax(unsigned int algo, double *x, uint64_t nx, ans_t *ans, int k, int align, double fill, bool narm, int hasna, bool verbose) {
+  if (nx < k) {
+    if (verbose)
+      snprintf(end(ans->message[0]), 500, _("%s: window width longer than input vector, returning all NA vector\n"), __func__);
```

Comment:
```suggestion
      snprintf(end(ans->message[0]), 500, _("%s: window width longer than input vector, returning all %d vector\n"), __func__, fill);
```

Reply:
int64 is not supported, only real..
`%d` format is for integer

Reply:
My bad thought I read smth about int64 support. changed to `%lf`